tests/test_auto/optuna/test_wrapper.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from optuna.pruners import MedianPruner
+from optuna.samplers import GridSampler
+from optuna.samplers import TPESampler
+from optuna.storages import InMemoryStorage
+from optuna.study import StudyDirection
+
+from etna.auto.optuna import Optuna
+
+
+@pytest.fixture()
```

Comment:
May be we should remove parenthesis.

Reply:
We have the same `()` in multiple places. Maybe we should add linters for example. Or make nothing